Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml

```
@@ -12,7 +12,7 @@
   id: ScarEyeLeft
   bodyPart: Head
   markingCategory: Head
-  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin] # DeltaV - Felinid-Vulpkanin
+  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin, Kitsune]  # DeltaV - Felinid-Vulpkanin-Kitsune
```

The `Felinid-Vulpkanin` meant "From Felinid to Vulpkanin"
```suggestion
  speciesRestriction: [Human, Dwarf, Felinid, Harpy, Oni, Rodentia, Feroxi, Vulpkanin, Kitsune]  # DeltaV - Felinid-Kitsune
```